website/data/adopters.yml

```diff
@@ -78,3 +78,9 @@ adopters:
 - name: Banzai Cloud
   url: https://banzaicloud.com/
   logo: banzaicloud.png
+- name: Jurumani Solutions
```

Comment:
unrelated to this PR??

Reply:
Sorry chained PRs problems -> Moved this PR to target master now since the base PR was merged. (: Now should be fine! Sorry.

Please PTAL